src/system.h

```diff
@@ -114,6 +117,8 @@ typedef struct sock_filter bpf_instr_raw;
 int sys_chk_seccomp_syscall(void);
 void sys_set_seccomp_syscall(bool enable);
 
+int sys_chk_seccomp_action(uint32_t action);
```

Comment:
I'm also wondering if we need a `sys_set_seccomp_action(...)` function.

Reply:
We do need one. It gets added in the `all: add support for new log action` patch.

Reply:
Ah ha, found it later in the patchset.  It probably should be included here, but not the end of the world.

Reply:
It sounds like you're not *requiring* the patchset surgery needed to move `sys_set_seccomp_action()` up a patch in the series so I'm going to opt for leaving it where it is at rather than risk making a mistake since we seem to be close to merging this. Let me know if that's a problem.